client/core/types.go

```diff
@@ -396,6 +403,12 @@ func coreOrderFromTrade(ord order.Order, metaData *db.OrderMetaData) *Order {
 		accelerationCoins = append(accelerationCoins, NewCoin(fromID, coinID))
 	}
 
+	// For in-flight orders, we'll set the order ID as a zero-hash.
```

Comment:
OK but will you comment why this is necessary instead of leaving nil? What specifically goes wrong if we don't do this?  Is it just for display?  Does a consumer recognize a zero hash vs zero length?

Reply:
Hmm, I think we can leave it empty. Instead of having to loop through every order ID string character.